.github/contributing.md

```diff
@@ -39,7 +39,7 @@ Issues labelled `good first contribution`.
 For your contribution to be accepted:
 
 - [x] You must have signed the [Contributor License Agreement (CLA)](https://cla-assistant.io/slackapi/python-slack-sdk).
-- [x] The test suite must be complete and pass.
+- [x] The test suite must be complete and pass (see the [Maintainer's Guide](./maintainers_guide.md) for details on how to run the tests).
```

Comment:
Haha I see you did do this (i just mentioned it as a comment on the issue thread)